src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Explore/Evaluation/index.js

```diff
@@ -38,13 +38,14 @@ class TCEvaluation extends Component {
           const previous_incorrect_dict = this.state.incorrect_classification
           previous_incorrect_dict[value] = "loading"
           this.setState({"incorrect_classification": previous_incorrect_dict})
-          if (window.navigator.platform == 'MacIntel') {
+          if (window.navigator.platform === 'MacIntel') {
             window.webkit.messageHandlers["scriptHandler"].postMessage({
               "status": 'getIncorrects',
               "label": value + ""
             });
           }
         }
+	return null;
```

Comment:
It's weird to have a `map` that returns null on every value, and then we throw away the result anyway. Instead of returning null here, we should probably use [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) instead, so it doesn't expect a value to be returned.